articles/migrations/guides/passwordless-start.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Migration Guide for the Use of /passwordless/start from Confidential Applications
+description: Auth0 is deprecating the usage of the /passwordless/start endpoint from confidential applications without a client secret in the request.
+topics:
+  - passwordless
+  - migrations
+contentType:
+  - concept
+  - how-to
+useCase:
+  - customize-connections
+---
+# Migration Guide: Use of /passwordless/start from Confidential Applications
+
+Auth0 is deprecating the use of the `/passwordless/start` endpoint from confidential applications when no client secret is included in the request.
+
+OAuth uses the term 'confidential' for applications that can store secrets. In Auth0, those are 'Regular Web Applications', which serve web pages from a backend app. Single Page Applications and Native Applications are considered 'public' applications, and are not affected by this change.
+
+## Does this affect me?
+
+If your applications currently calls the `/passwordless/start` endpoint directly to begin passwordless authentication from a Web Application, this deprecation does affect you.
+
+You can verify if that is the case by checking the [tenant logs](${manage_url}/#/logs), filtering by "Deprecation Notice" and check for logs saying "Enforce client authentication for passwordless connections".
```

Comment:
We should provide a copy/paste query to check for use. See https://auth0.com/docs/logs/migrate-logs-v2-v3#how-can-i-check-to-see-if-i-ve-migrated-all-my-queries- for a good example. That page also includes an explanation of how ofter deprecation notices are generated. If that applies here we should include that as well.

Reply:
Also, the deprecation notice should include information to help determine which of the changes below need to be made. We should include that information here.